tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/execution_data/execution_data_component.css

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+.debug-tensor-values-container {
+  height: 60px;
+  max-height: 80px;
+  overflow-x: auto;
+  overflow-y: auto;
+}
+
+.debug-tensor-values-table {
+  width: 100%;
+}
+
+.debug-tensor-values-table td {
+  border-top: 1px solid black;
+  text-align: left;
+}
+
+.debug-tensor-values-table th {
+  text-align: left;
+}
+
+.execution-data-field {
+  white-space: nowrap;
+}
+
+.execution-data-key {
+  display: inline-block;
+  max-width: 120px;
```

Comment:
I assume it is because descendent can take up width larger than 120px? If so, would you want to `overflow: hidden` them (and put ellipsis on the text)?